src/dynamicTypes/typesLoader.ts

```diff
@@ -24,10 +24,21 @@ const dynamicallyImportFullCorvidTypes = async (): Promise<{
   );
 };
 
+const dynamicallyImportFullCorvidTypesDeprecated = async (): Promise<{
+  default: FullCorvidTypes;
+}> => {
+  return import(
+    /* webpackChunkName: "fullCorvidTypesJSON.deprecated" */ "../../dist/fullCorvidTypes.deprecated.json"
+  );
+};
+
 export const getCollectionLibs = async (
-  key: LibCollections
+  key: LibCollections,
+  { multipleFiles }: { multipleFiles?: boolean } = {}
```

Comment:
Why do it in an object? I think it's more readable if outside, but your call.
Anyway, cosider extracting into a type and giving a default value.
```suggestion
  { multipleFiles }: SomeType = {multipleFiles: false}
```